src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -334,21 +362,18 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                   <ScrollableContainer maxHeight={300}>
                     <ul className="govuk-list">
                       {[
-                        ...otherPublicationReleases.map(
-                          ({ id, slug, title }) => (
-                            <li key={id} data-testid="other-release-item">
-                              <Link
-                                to={`/find-statistics/${release.publication.slug}/${slug}`}
-                              >
-                                {title}
-                              </Link>
-                            </li>
-                          ),
-                        ),
-                        ...release.publication.legacyReleases.map(
-                          ({ id, description, url }) => (
```

Comment:
There is code like this in Admin's `ReleaseContent.tsx` that also needs updating.

I think probably some legacy release stuff that can be removed from view models when this is done soon, like from `publicationService`'s `interface Publication`.